Makefile.toml

```diff
@@ -38,13 +38,15 @@ echo success
 [tasks.pre-dev]
 category = "Develop"
 description = "Downloads necessary tools and build required components."
-dependencies = ["download-minio", "download-mcli", "clean-data"]
+dependencies = ["download-minio", "download-mcli", "download-prometheus", "download-grafana", "clean-data"]
 script = '''
 #!/bin/bash
 # run minio
 tmux new -d -s runkv-minio ${PREFIX_BIN}/minio server --address 127.0.0.1:9000 --console-address 127.0.0.1:9090 ${PREFIX_DATA}
 # run jaeger
 tmux new -d -s runkv-jaeger ${PREFIX_BIN}/jaeger
+# run prometheus
+tmux new -d -s runkv-prometheus "${PREFIX_BIN}/prometheus/prometheus" --config.file="${PWD}/tests/etc/prometheus.yml" --web.listen-address=:9091 --storage.tsdb.path="${PREFIX_DATA}/prometheus_data"
```

Comment:
`$PWD` should be removed. Or if we run `run` from other directory, it will refer to wrong directory. 🤔